tests/load/bigquery/test_bigquery_client.py

```diff
@@ -300,14 +277,23 @@ def test_bigquery_job_errors(client: BigQueryClient, file_storage: FileStorage)
         "timestamp": str(pendulum.now()),
     }
     job = expect_load_file(client, file_storage, json.dumps(load_json), user_table_name)
+    assert job._created_job  # type: ignore
 
     # start a job from the same file. it should be a fallback to retrieve a job silently
-    r_job = client.start_file_load(
-        client.schema.get_table(user_table_name),
-        file_storage.make_full_path(job.file_name()),
-        uniq_id(),
+    r_job = cast(
+        RunnableLoadJob,
+        client.get_load_job(
+            client.schema.get_table(user_table_name),
+            file_storage.make_full_path(job.file_name()),
+            uniq_id(),
+        ),
     )
+
+    # job will be automatically found and resumed
```

Comment:
this test is not testing any resumes because `get_load_job` does not create anything in bigquery. you'll need to create a job that in big query, make sure it exists and then complete it. how to do that exactly IDK.

it would be cool to test it inside `load.py`. so you load a job that fails once you create another load instance run it again and make sure it is completed with retry count == 1 in file name

Reply:
I call "r_job.run_managed(client)" here which actually connects to bigquery and resumes the job (as indicated by the _resumed_job boolean var). The test is as good as the tests that where there for bigquery before. I am happy to improve them, but I don't really want to bloat the scope of these changes with things are are not directly related, at least in my view.